Fails to build with OpenCL 3.0 headers

Trying to build with `python3 setup.py build` i get this:

```
x86_64-linux-gnu-gcc -Wno-unused-result -Wsign-compare -fwrapv -Wall -O3 -DNDEBUG -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FO>
In file included from /home/michal/0/build/b_pocl_master/examples/PyOpenCL/src/PyOpenCL/.eggs/numpy-1.26.1-py3.10-linux-x86_64.egg/numpy/core/include/numpy/n>
                 from /home/michal/0/build/b_pocl_master/examples/PyOpenCL/src/PyOpenCL/.eggs/numpy-1.26.1-py3.10-linux-x86_64.egg/numpy/core/include/numpy/n>
                 from /home/michal/0/build/b_pocl_master/examples/PyOpenCL/src/PyOpenCL/.eggs/numpy-1.26.1-py3.10-linux-x86_64.egg/numpy/core/include/numpy/a>
                 from src/wrap_cl.hpp:101,
                 from src/wrap_constants.cpp:30:
/home/michal/0/build/b_pocl_master/examples/PyOpenCL/src/PyOpenCL/.eggs/numpy-1.26.1-py3.10-linux-x86_64.egg/numpy/core/include/numpy/npy_1_7_deprecated_api.>
   17 | #warning "Using deprecated NumPy API, disable it with " \
      |  ^~~~~~~
src/wrap_constants.cpp: In function ‘void pyopencl_expose_constants(pybind11::module_&)’:
src/wrap_constants.cpp:158:21: error: ‘CL_UNORM_INT24’ was not declared in this scope; did you mean ‘CL_UNORM_INT8’?
  158 |   cls.attr(#NAME) = CL_##PREFIX##NAME
      |                     ^~~
src/wrap_constants.cpp:754:5: note: in expansion of macro ‘ADD_ATTR’
  754 |     ADD_ATTR( , UNORM_INT24);
      |     ^~~~~~~~
error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
```

The problem is that `CL_UNORM_INT24` is only available if `cl_gl.h` has been included., so it seems the code is missing some ifdef.

--- a/src/wrap_constants.cpp
+++ b/src/wrap_constants.cpp
@@ -750,7 +750,7 @@ void pyopencl_expose_constants(py::module_ &m)
     ADD_ATTR( , UNSIGNED_INT32);
     ADD_ATTR( , HALF_FLOAT);
     ADD_ATTR( , FLOAT);
-#if PYOPENCL_CL_VERSION >= 0x1020
+#if PYOPENCL_CL_VERSION >= 0x1020 && defined(cl_khr_gl_sharing)
     ADD_ATTR( , UNORM_INT24);
 #endif
 #if PYOPENCL_CL_VERSION >= 0x2010

